kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1231,6 +1231,22 @@ object KyuubiConf {
       .checkValue(_ > 0, "must be positive number")
       .createWithDefault(Duration.ofMinutes(5).toMillis)
 
+  val KUBERNETES_APPLICATION_STATE_FROM_CONTAINER: ConfigEntry[Boolean] =
+    buildConf("kyuubi.kubernetes.application.state.fromContainer")
+      .doc("If set to true then the application state will be retrieved from the container " +
+        "instead of the pod.")
+      .version("1.8.1")
+      .booleanConf
+      .createWithDefault(false)
+
+  val KUBERNETES_APPLICATION_STATE_CONTAINERS: ConfigEntry[Set[String]] =
+    buildConf("kyuubi.kubernetes.application.state.containers")
+      .doc("A comma-separated list of container names to retrieve the application state from.")
+      .version("1.8.1")
+      .stringConf
+      .toSet()
+      .createWithDefault(Set.empty)
```

Comment:
how about set default to `spark-kubernetes-driver`(default spark driver container name)

Reply:
SGTM